dashboard/app/serializers/api/v1/pd/session_attendance_serializer.rb

```diff
@@ -16,6 +16,7 @@ def attendance
         email: enrollment.email,
         enrollment_id: enrollment.id,
         user_id: id_or_nil(enrollment.user),
+        verified_teacher_account: !!User.find_by_id(enrollment.user.try(:id)).try(:authorized_teacher?),
```

Comment:
1. Why look up the user by id from the id of the user we already have (my head hurts)? Why not just inspect `enrollment.user` directly?

 2. Since we have for some time been confusingly referring to `authorized_teachers` as `verified teachers` and this is now be codified and made official, how about aliasing `User.authorized_teacher?` as `User.verified_teacher?` to alleviate the confusion?

Then this line can be simplified to: 
```ruby
verified_teacher_account: !!enrollment.user.try(:verified_teacher?)
```